canton/BUILD.bazel

```diff
@@ -39,11 +39,11 @@ cat << EOF > $@
 package com.digitalasset.canton.buildinfo
 
 case object BuildInfo {{
-  val version: String = "2.7.0-SNAPSHOT"
+  val version: String = "3.0.0-SNAPSHOT"
   val scalaVersion: String = "{scala_version}"
   val sbtVersion: String = "bazel"
   val damlLibrariesVersion: String = "{sdk_version}"
-  val protocolVersions = List("3", "4", "5")
+  val protocolVersions = Seq()
```

Comment:
Yes, it is currently empty in the file generated by sbt, which we reproduce here.

Reply:
The reason is that protocol versions listed here should be stable but the current one (30) is unstable.
Note: I think we should find a way for you to read our file rather than hardcoding the values